### PR TITLE
[TR] Update Mascots/Gallery

### PR DESCRIPTION
**Pull request by zentoryny**
updated version

### CODE REVIEW

File: wiki/Mascots/Gallery/tr.md [hunk 1]
```diff
@@ -1,42 +1,111 @@
...
 ![404 hatası resmi.](img/pippi-Daru-404.png "404 hatası resmi.")
 
+#### Fan tasarımları
+
+Pippi için fan tasarımları:
+
+- [İlk Resmi osu! Fanart Yarışması Sonuçları! (Facebook)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152186520298282)
+- [osu! Fanart Yarışması 2! (Facebook)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152291754173282)
+- [osu! Fanart Yarışması 3! (Facebook)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152514671088282)
+- [Fan Art 7: Halloween!](https://osu.ppy.sh/community/contests/5)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:38]:
```suggestion
- [Fan Art 7: Cadılar Bayramı!](https://osu.ppy.sh/community/contests/5)
```

File: wiki/Mascots/Gallery/tr.md [hunk 2]
```diff
@@ -1,42 +1,111 @@
...
+- [Fan Art 8: Christmas](https://osu.ppy.sh/community/contests/36)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:39]:
```suggestion
- [Fan Art 8: Noel](https://osu.ppy.sh/community/contests/36)
```

File: wiki/Mascots/Gallery/tr.md [hunk 3]
```diff
@@ -1,42 +1,111 @@
...
 ![osu!catch'te Kiai'de Yuzu](img/Yuzu-kiai.png "osu!catch'te Kiai'de Yuzu")
+
 ![osu!catch'te meyve kaçırınca Yuzu](img/Yuzu-fail.png "osu!catch'te meyve kaçırınca Yuzu")
 
+#### Fan tasarımları
+
+Yuzu için fan tasarımları:
+
+- [[osu! Fanart Yarışması 3! (Facebook)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152514671088282)
+- [Fan Art 7: Halloween!](https://osu.ppy.sh/community/contests/5)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:57]:
```suggestion
- [Fan Art 7: Cadılar Bayramı!](https://osu.ppy.sh/community/contests/5)
```

File: wiki/Mascots/Gallery/tr.md [hunk 4]
```diff
@@ -1,42 +1,111 @@
...
+- [Fan Art 8: Christmas](https://osu.ppy.sh/community/contests/36)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:58]:
```suggestion
- [Fan Art 8: Noel](https://osu.ppy.sh/community/contests/36)
```

File: wiki/Mascots/Gallery/tr.md [hunk 5]
```diff
@@ -1,42 +1,111 @@
...
+
+### Maria
+
+![Pippi ve maria ile fan tasarımı](img/Maria-fanart-1.jpg "Fanart by Anny")
+
+#### Fan Tasarımları
+
+Daha fazla maria fan tasarımları:
+
+- [Fan Art 8: Christmas](https://osu.ppy.sh/community/contests/36)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:69]:
```suggestion
- [Fan Art 8: Noel](https://osu.ppy.sh/community/contests/36)
```

File: wiki/Mascots/Gallery/tr.md [hunk 6]
```diff
@@ -1,42 +1,111 @@
...
+- [Fan Art 9: Spring](https://osu.ppy.sh/community/contests/45)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:70]:
```suggestion
- [Fan Art 9: İlkbahar](https://osu.ppy.sh/community/contests/45)
```

File: wiki/Mascots/Gallery/tr.md [hunk 7]
```diff
@@ -1,42 +1,111 @@
...
+
+Cameo Maskotları için fan tasarımları:
+
+- [Mocha Yaz Fanart Yarışması'nda](https://osu.ppy.sh/community/contests/48)
+
+## Topluluk
+
+Topluluk yapımı fan tasarımı maskotlar:
+
+- [Fan Art 6: osu!taiko mascot](https://osu.ppy.sh/community/contests/2)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:106]:
```suggestion
- [Fan Art 6: osu!taiko maskotu](https://osu.ppy.sh/community/contests/2)
```

File: wiki/Mascots/Gallery/tr.md [hunk 8]
```diff
@@ -1,42 +1,111 @@
...
+- [Fan Art 7: Halloween!](https://osu.ppy.sh/community/contests/5)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:107]:
```suggestion
- [Fan Art 7: Cadılar Bayramı!](https://osu.ppy.sh/community/contests/5)
```

File: wiki/Mascots/Gallery/tr.md [hunk 9]
```diff
@@ -1,42 +1,111 @@
...
+- [Fan Art 8: Christmas](https://osu.ppy.sh/community/contests/36)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:108]:
```suggestion
- [Fan Art 8: Noel](https://osu.ppy.sh/community/contests/36)
```

File: wiki/Mascots/Gallery/tr.md [hunk 10]
```diff
@@ -1,42 +1,111 @@
...
+- [Fan Art 9: Spring](https://osu.ppy.sh/community/contests/45)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:109]:
```suggestion
- [Fan Art 9: İlkbahar](https://osu.ppy.sh/community/contests/45)
```

**Comment by cl8n** [comment]
I think it's fine to have English if it's the actual title of something (in this case, all the contests)

**Comment by cl8n** [comment]
oh I see what you meant now, that they're mixed between languages

yeah it should either be original title only, or something like `Translated title (*Original title*)`

File: wiki/Mascots/Gallery/tr.md [hunk 11]
```diff
@@ -1,42 +1,111 @@
...
 
+#### Fan tasarımları
+
+Pippi için fan tasarımları:
+
+- [İlk Resmi osu! Fanart Yarışması Sonuçları! (*First Official osu! Fanart Contest Results!*)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152186520298282 "Facebook")
+- [osu! Fanart Yarışması 2! (*osu! Fanart Contest 2!*)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152291754173282 "Facebook")
+- [osu! Fanart Yarışması 3! (*osu! Fanart Contest 3!*)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152514671088282 "Facebook")
+- [Fan Art 7: Halloween!](https://osu.ppy.sh/community/contests/5)
+- [Fan Art 8: Christmas](https://osu.ppy.sh/community/contests/36)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:39]:
```suggestion
- [Fan Art 7: Cadılar Bayramı! (*Fan Art 7: Halloween!*)](https://osu.ppy.sh/community/contests/5)
- [Fan Art 8: Noel (*Fan Art 8: Christmas*)](https://osu.ppy.sh/community/contests/36)
```

File: wiki/Mascots/Gallery/tr.md [hunk 12]
```diff
@@ -1,42 +1,111 @@
...
+
 ![osu!catch'te meyve kaçırınca Yuzu](img/Yuzu-fail.png "osu!catch'te meyve kaçırınca Yuzu")
 
+#### Fan tasarımları
+
+Yuzu için fan tasarımları:
+
+- [osu! Fanart Yarışması 3! (*osu! Fanart Contest 3!*)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152514671088282 "Facebook")
+- [Fan Art 7: Halloween!](https://osu.ppy.sh/community/contests/5)
+- [Fan Art 8: Christmas](https://osu.ppy.sh/community/contests/36)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:58]:
```suggestion
- [Fan Art 7: Cadılar Bayramı! (*Fan Art 7: Halloween!*)](https://osu.ppy.sh/community/contests/5)
- [Fan Art 8: Noel (*Fan Art 8: Christmas*)](https://osu.ppy.sh/community/contests/36)
```

File: wiki/Mascots/Gallery/tr.md [hunk 13]
```diff
@@ -1,42 +1,111 @@
...
+### Maria
+
+![Pippi ve maria ile fan tasarımı](img/Maria-fanart-1.jpg "Fanart by Anny")
+
+#### Fan Tasarımları
+
+Daha fazla maria fan tasarımları:
+
+- [Fan Art 8: Christmas](https://osu.ppy.sh/community/contests/36)
+- [Fan Art 9: Spring](https://osu.ppy.sh/community/contests/45)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:70]:
```suggestion
- [Fan Art 8: Noel (*Fan Art 8: Christmas*)](https://osu.ppy.sh/community/contests/36)
- [Fan Art 9: İlkbahar (*Fan Art 9: Spring*)](https://osu.ppy.sh/community/contests/45)
```

File: wiki/Mascots/Gallery/tr.md [hunk 14]
```diff
@@ -1,42 +1,111 @@
...
+- [Mocha Yaz Fanart Yarışması'nda (*Mocha in Summer Fanart Contest*)](https://osu.ppy.sh/community/contests/48)
+
+## Topluluk
+
+Topluluk yapımı fan tasarımı maskotlar:
+
+- [Fan Art 6: osu!taiko mascot](https://osu.ppy.sh/community/contests/2)
+- [Fan Art 7: Halloween!](https://osu.ppy.sh/community/contests/5)
+- [Fan Art 8: Christmas](https://osu.ppy.sh/community/contests/36)
+- [Fan Art 9: Spring](https://osu.ppy.sh/community/contests/45)
```
Comment (zeusminus) [inline wiki/Mascots/Gallery/tr.md:109]:
```suggestion
- [Fan Art 6: osu!taiko maskotu (*Fan Art 6: osu!taiko mascot*)](https://osu.ppy.sh/community/contests/2)
- [Fan Art 7: Cadılar Bayramı! (*Fan Art 7: Halloween!*)](https://osu.ppy.sh/community/contests/5)
- [Fan Art 8: Noel (*Fan Art 8: Christmas*)](https://osu.ppy.sh/community/contests/36)
- [Fan Art 9: Spring](https://osu.ppy.sh/community/contests/45)
```

Comment (TPGPL) [inline wiki/Mascots/Gallery/tr.md:109]:
```suggestion
- [Fan Art 9: İlkbahar (*Fan Art 9: Spring*)](https://osu.ppy.sh/community/contests/45)
```

**Pull request merged**